verificarlo.in.in

```diff
@@ -44,61 +44,66 @@ clang = '@CLANG_PATH@'
 opt = llvm_bindir + '/opt'
 dragonegg = "@DRAGONEGG_PATH@"
 gcc = "@GCC_PATH@"
-FORTRAN_EXTENSIONS=[".f", ".f90", ".f77"]
-C_EXTENSIONS=[".c", ".cpp", ".cc", ".cxx"]
+FORTRAN_EXTENSIONS = [".f", ".f90", ".f77"]
+C_EXTENSIONS = [".c", ".cpp", ".cc", ".cxx"]
+
 
 class NoPrefixParser(argparse.ArgumentParser):
     # ignore prefix autocompletion of options
     def _get_option_tuples(self, option_string):
         return []
 
+
 def fail(msg):
     print(sys.argv[0] + ': ' + msg, file=sys.stderr)
     sys.exit(1)
 
+
 def is_fortran(name):
     return os.path.splitext(name)[1].lower() in FORTRAN_EXTENSIONS
 
+
 def is_c(name):
     return os.path.splitext(name)[1].lower() in C_EXTENSIONS
 
+
 def parse_extra_args(args):
     sources = []
     options = []
 
     for a in args:
         if is_fortran(a):
-	    if not dragonegg:
-		fail("fortran not supported. "
-		     + "--without-fortran was used during configuration.")
-	    sources.append(a)
-	elif is_c(a):
-            sources.append(a)
+            if not dragonegg:
+                fail("fortran not supported. "
+                     + "--without-fortran was used during configuration.")
+                sources.append(a)
+            elif is_c(a):
```

Comment:
When reindenting the `elif` has not the correct parent anymore.

Reply:
Good catch - thanks. That was sloppy on my part, was just trying to not have both tabs+spaces in the same file.